e2e/cypress/integration/02-apd/04-apd-budget-checks.cy.js

```diff
@@ -352,12 +352,12 @@ describe('APD with Data', { tags: ['@apd', '@data', '@slow'] }, () => {
       checkProposedBudget(
         years,
         budgetData.afterStateExpenseNoFFP,
-        [553250, 557750],
+        [1173674, 1311620],
         'noFFP'
       );
     });
 
-    it('Checks Private Contractors Budget', () => {
+    it.skip('Checks Private Contractors Budget', () => {
```

Comment:
do we need to still skip this?

Reply:
still had to do this section at the time. but with my most recent commit I added it back in